tests/suit/framework/framework.go

```diff
@@ -171,7 +171,7 @@ func (f *Framework) CreateNamespace(prefix string, labels map[string]string) (*v
 	}
 	// TODO: restore the pod-security label once https://github.com/kubev2v/forklift/issues/173 is fixed
 	// pod-security.kubernetes.io/<MODE>: <LEVEL>
-	//labels["pod-security.kubernetes.io/enforce"] = "restricted"
+	labels["pod-security.kubernetes.io/enforce"] = "restricted"
```

Comment:
as we spoke offline - this annotation can be reintroduced after changing the conversion pod (virt-v2v)